sigs.yaml

```diff
@@ -1,4 +1,42 @@
 sigs:
+- dir: sig-community-experience
+  name: Community Experience
+  mission_statement: >
+    Marketing/Communications/Org-Ops/Governance/Community owns O2KR2 “Communications”
+    and “Metrics” owns O2KR1 “all workstreams / people”
```

Comment:
As KRs are relatively short-lived (compared to SIGs), I think it's better to leave them out of the mission statement. We don't want a mission statement subject to change every few months, right?

Reply:
redefined in one sentence: Our goal is to nurture a sustainable community by stabilizing and growing it through delivering good quality services using the data driven approach.